api/users.go

```diff
@@ -136,17 +136,23 @@ func (api *API) UpdateUserHandler(ctx context.Context, w http.ResponseWriter, re
 		return nil, models.NewErrorResponse(http.StatusBadRequest, nil, validationErrs...)
 	}
 
-	userRequest := user.BuildUpdateUserRequest(api.UserPoolId)
+	if user.Active {
+		userEnableRequest := user.BuildEnableUserRequest(api.UserPoolId)
```

Comment:
can UserPoolId be null or empty?

Reply:
No I don't think it can be.
When the API is setup the userPoolId is set (along with some other required config) and if the value isn't there is throws and error and shuts down.
https://github.com/ONSdigital/dp-identity-api/blob/develop/api/api.go#L52